portal-impl/src/com/liferay/portal/upgrade/v7_4_x/PortalUpgradeProcessRegistryImpl.java

```diff
@@ -35,6 +35,8 @@ public void registerUpgradeProcesses(
 		upgradeProcesses.put(new Version(9, 0, 1), new UpgradeModules());
 
 		upgradeProcesses.put(new Version(9, 1, 0), new UpgradeCountry());
+
+		upgradeProcesses.put(new Version(9, 1, 1), new UpgradeSchema());
```

Comment:
Looks like this will have to change. In all other examples, `UpgradeSchema` marks a major version bump, then all others have to come after it.

See `git grep --fixed-strings --function-context 'new UpgradeSchema()' -- portal-impl`